src/library/library_prefs.cpp

```diff
@@ -48,6 +48,11 @@ const ConfigKey mixxx::library::prefs::kSearchDebouncingTimeoutMillisConfigKey =
                 mixxx::library::prefs::kConfigGroup,
                 QStringLiteral("SearchDebouncingTimeoutMillis")};
 
+const ConfigKey mixxx::library::prefs::kSearchBpmFuzzyRangeConfigKey =
+        ConfigKey{
+                mixxx::library::prefs::kConfigGroup,
+                QStringLiteral("search_bpm_fuzzy_range")};
```

Comment:
We have "SearchDebouncingTimeoutMillis" and "search_bpm_fuzzy_range" 
Do we want to follow the first in this case?

Reply:
I thought we want to migrate to snake case for ConfigKeys?

Reply:
I don't mind. I just wanted to mention it.